manuscript/main.tex

```diff
@@ -681,9 +681,9 @@ \subsubsection*{Gut microbial taxa are associated with specific cognitive functi
 Mullen Scales of Early Learning (MSEL, birth to 36 months - \textit{see Methods}). 
 Taxonomic profiles coupled with demographics exhibited predictive potential towards three
 of five probed subscales (Figure 4A), 
-namely Expressive Language ( test-set RMSE \hl{XXX}, test-set correlation \hl{YYYY}), 
-Gross Motor (test-set RMSE \hl{XXX}, test-set correlation \hl{YYYY}) and 
-Visual Reception (test-set RMSE \hl{XXX}, test-set correlation \hl{YYYY}).
+namely Expressive Language ( test-set RMSE \hl{12.05}, test-set correlation \hl{0.368}), 
+Gross Motor (test-set RMSE \hl{11.07}, test-set correlation \hl{0.137}) and 
```

Comment:
Can you remove the `\hl{...}` once they're fixed?

Reply:
Nevermind, I took care of it. Make sure to `git pull` before continuing to work on this branch